packages/common/src/ui/layout/tables/components/Cells/CurrencyInputCell/CurrencyInputCell.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { CellProps } from '../../../columns';
+import { CurrencyInput } from '../../../../../components/inputs';
+import { DomainObject } from '../../../../../../types';
+import { useDebounceCallback } from '../../../../../../hooks';
+
+type R<T> = T &
+  DomainObject & { update?: (key: string, value: string) => void };
+
+type CP<T> = CellProps<R<T>>;
+
+export const CurrencyInputCell = <T extends DomainObject>({
+  rowData,
+  column,
+}: CP<T>): React.ReactElement<CP<T>> => {
+  const [buffer, setBuffer] = React.useState(Number(column.accessor(rowData)));
+
+  const noop = () => {};
+  const updater = useDebounceCallback(rowData.update ?? noop, [rowData], 250);
```

Comment:
im not sure about the debounce. It's helpful for not going crazy on the rendering but does it actually make things seem laggier?

Reply:
I'm not noticing any lag myself